tslearn/clustering.py

```diff
@@ -227,6 +246,9 @@ class GlobalAlignmentKernelKMeans(BaseEstimator, ClusterMixin):
         Bandwidth parameter for the Global Alignment kernel. If set to 'auto',
         it is computed based on a sampling of the training set
         (cf :ref:`tslearn.metrics.sigma_gak <fun-sigmagak>`)
+    n_jobs : int or None (default: None)
```

Comment:
Perhaps a small note could be made that -1 will use all available cores?

Reply:
Agree, it is mentioned in scikit-learn (see [sklearn.ensemble.RandomForestClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html) for instance)